backend/src/users/user/user.controller.ts

```diff
@@ -73,6 +73,13 @@ export class UserController {
 		});
 	}
 
+	// @UseGuards(AuthGuard)
+	// // localhost:8080/api/users/id/games
+	// @Get(':id/games')
+	// async findOneWithGame(@Param('id') id: number): Promise<Game[]> {
+	// 	return this.userService.getGames(id);
+	// }
+
```

Comment:
This sneaked back in as well, maybe with the pull from the main will resolve the last things changed.

Reply:
this is probably just me sloppily merging main into this branch and picking the wrong option...